hw03_vesselsj2.py

```diff
@@ -0,0 +1,152 @@
+#################################################################################
+# Author: Jaren Vessels
+# Username: vesselsj2
+#
+# Assignment: T03: Boustrophedon Turtles
+# Purpose: To draw an illusion with a person floating on it
+#################################################################################
+# Acknowledgements:
+#   Runestone Academy Chapter 5: Modules
+#   Used AI to help debug random module that was not letting me randomize the color of my screen and turtles
+#   Used AI to help me debug my illusion pattern that was not increasing in size on each repetition
+#################################################################################
+
+import turtle
+import random
+
+
+def ran_col():
+    """
+    Creates the random rgb generator for my code
+    """
+    r = int(random.random() * 255)
+    g = int(random.random() * 255)
+    b = int(random.random() * 255)
+    return r, g, b
+
+
+def draw_person(ben, wn):
+    wn.bgcolor(ran_col())
```

Comment:
docstrings should be the first line after the function definition. Move `wn.bgcolor(ran_col())` to below the docstring.